app/static/css/styles.css

```
@@ -178,4 +178,212 @@ a.logout-btn:hover, a.logout-btn:focus, a.logout-btn:active {
+}
+
+.consent-section p,
+.consent-section li {
+    color: #d5d7dc;
+    line-height: 1.65;
+}
+
+.consent-section a {
+    color: var(--th-yellow);
+    text-decoration: none;
+    word-break: break-word;
```

Stylelint (17.7.0)

[error] Deprecated keyword "break-word" for property "word-break" (declaration-property-value-keyword-no-deprecated)

(declaration-property-value-keyword-no-deprecated)